addons/plugin_refresher/refresh_button.gd

```diff
@@ -87,7 +87,6 @@ func _background_check():
 		get_tree().create_timer(background_check_cycle_sec).timeout.connect(_background_check)
 
 func _load_settings():
-	var current_main_screen = null
```

Comment:
The local variable "current_main_screen" is shadowing an already-declared variable at line 101.

(This variable seems to be unused, so I removed it.)